src/main/java/uk/nhs/cactus/common/elasticsearch/ElasticSearchRoleClient.java

```diff
@@ -0,0 +1,61 @@
+package uk.nhs.cactus.common.elasticsearch;
+
+import static org.apache.http.HttpHeaders.CONTENT_TYPE;
+import static org.apache.http.entity.ContentType.APPLICATION_JSON;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.http.Header;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpHost;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.entity.ByteArrayEntity;
+import org.apache.http.impl.client.BasicResponseHandler;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.message.BasicHeader;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+import uk.nhs.cactus.common.elasticsearch.role.PutRoleMappingRequest;
+import uk.nhs.cactus.common.elasticsearch.role.PutRoleRequest;
+
+@Component
+@Slf4j
+@Profile("!dev")
+@RequiredArgsConstructor
+public class ElasticSearchRoleClient {
```

Comment:
Agreeing with Charlie: presumably this is only used in the EMS, but doesn't hurt to be here.

Reply:
Yep, indeed, I've moved it out.